LSP mode doesn't work on Windows (0.8.0)

Using plugin version 0.8.0, and tried using LSP mode, and it completely fails outright. From the looks of the output to the console, it appears it's passing a malformed path to the server:

```
[2018-04-25 08:46:31.636] [ERROR] nuclide-flow-rpc - Flow failed: flow get-def --json --path c%3A\Users\sargant\Repositories\Customer\Project\Source\Web\ClientApp\src\App.js 39 60.
```

The large JSON error block that follows is huge, but the first few lines seem pertinent:

```json
{
  "name": "ProcessExitError",
  "exitCode": 12,
  "signal": null,
  "stderr": "Could not find file or directory c:\\Users\\sargant\\Repositories\\Customer\\Project\\Source\\Web\\ClientApp\\c%3A\\Users\\sargant\\Repositories\\Customer\\Project\\Source\\Web\\ClientApp\\src\\App.js; canceling search for .flowconfig.\r\nSee \"flow init --help\" for more info\r\n", 
  ...snip...
}
```

Looks like arelative/absolute path problems with Windows paths. Can't tell if this is the responsibility of this plugin or the language server though.

--- a/lib/flowLSP.js
+++ b/lib/flowLSP.js
@@ -63,6 +63,9 @@ export function activate(context: ExtensionContext) {
       // Notify the server about file changes to '.clientrc files contain in the workspace
       fileEvents: workspace.createFileSystemWatcher('**/*.{js,jsx,mjs,js.flow}'),
     },
+    uriConverters: {
+      code2Protocol: uri => uri.toString(true), // this disables URL-encoding for file URLs
+    },
   };
 
   const statusBarItem = window.createStatusBarItem(StatusBarAlignment.Left, 0);

